stable/fluentd/templates/deployment.yaml

```diff
@@ -94,9 +98,19 @@ spec:
           configMap:
             name: {{ template "fluentd.fullname" . }}
         - name: buffer
-          {{- if .Values.persistence.enabled }}
+          {{- if and .Values.persistence.enabled (not .Values.autoscaling.enabled) }}
           persistentVolumeClaim:
             claimName: {{ template "fluentd.fullname" . }}
+          {{- else if and .Values.persistence.enabled (.Values.autoscaling.enabled) }}
+          volumeClaimTemplates:
+          - metadata:
+              name: buffer
```

Comment:
name: `{{ template "fluentd.fullname" . }}`

Reply:
"name: buffer", is the current configuration of the deployment.yml, we would like to change it?

Reply:
let's keep it as it is.